### ISSUE TITLE
Setup profile page

### ISSUE THREAD

**Issue by timarney** [cds-snc/platform-forms-client#2413]
https://www.figma.com/file/89fRce4gmy64UIArSZoAM8/v3.-GC-Forms-(builder-and-vault)?type=design&node-id=1908-63240&mode=design&t=VQ8JIJSa10IrJcc1-0

Setup the base layout for the "profile page"
Edit links to be handled in a separate task
Security questions etc... can be placeholder text

**Comment by samsadasivan** [comment]
Yay!

**Comment by timarney** [comment]
Merged which unblocks parts of https://github.com/cds-snc/platform-forms-client/issues/2418

PR: https://github.com/cds-snc/platform-forms-client/pull/2416
